application/src/main/kotlin/org/gxf/crestdeviceservice/command/service/CommandService.kt

```diff
@@ -33,6 +33,13 @@ class CommandService(
         if (deviceHasSameCommandAlreadyInProgress(command.deviceId, command.type)) {
             throw CommandValidationException("A command of the same type is already in progress.")
         }
+
+        if (command.type == Command.CommandType.ANALOG_ALARM_THRESHOLDS) {
+            val regex = Regex("([34]):-*(\\d+),-*(\\d+),-*(\\d+),-*(\\d+),-*(\\d+)")
```

Comment:
Consider:
```suggestion
            val regex = Regex("([34]):(-?\\d+),(-?\\d+),(-?\\d+),(-?\\d+),(-?\\d+)")
```
to make `-` an optional character that can appear only once and include it in the groups.